<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>about_window</class>
 <widget class="QDialog" name="about_window">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_varsion">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Парсер v{get_version()}</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_dev">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Разработал  © Зарихин В. А., 2022</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
